Error: Cannot find module 'lodash/pickBy'

I get this error after upgrading to 2.23.0: `Error: Cannot find module 'lodash/pickBy'`

Presumably related to https://github.com/adamgibbons/ics/pull/150?

```
"trace": [
        "Runtime.ImportModuleError: Error: Cannot find module 'lodash/pickBy'",
        "Require stack:",
        "- /var/task/node_modules/ics/dist/pipeline/build.js",
        "- /var/task/node_modules/ics/dist/pipeline/index.js",
        "- /var/task/node_modules/ics/dist/index.js",
        "- /var/task/node_modules/ics/index.js",
```

--- a/src/pipeline/build.js
+++ b/src/pipeline/build.js
@@ -1,5 +1,3 @@
-import pickBy from "lodash/pickBy";
-import identity from "lodash/identity";
 import defaultAttributes from "../defaults";
 
 export default function buildEvent(attributes = {}) {
@@ -32,7 +30,8 @@ export default function buildEvent(attributes = {}) {
   const output = Object.assign({}, defaultAttributes, attributes);
 
   // remove falsey values
-  const cleanOutput = pickBy(output, identity);
-
-  return cleanOutput;
+  return Object.entries(output).reduce(
+      (clean, entry) => entry[1] ? Object.assign(clean, {[entry[0]]: entry[1]}) : clean,
+      {}
+  )
 }

